### PR TITLE
Implemented Covalent API; Updated ptri_fees_base.py

### PR DESCRIPTION
**Pull request by specialist0x**
Currently, we need to update the `ptri_fees_base.py` when a new pool is added.  This PR implements covalent API to aggregate the top fee-generating pools, and publish them on our CDN.

- Implemented GCC flow to query covalent API for top fee-generating pools
- Output is uploaded to https://cdn.trisolaris.io/pools.json
- Updated `ptri_fees_base.py` to use `pools.json` response

### CODE REVIEW

File: gcc_utils.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def get_google_cloud_storage_blob(gcc_bucket, gcc_file_path):
     return blob
 
 
-def gccPrint(*message, severity="DEFAULT"):
+def gccPrint(message, severity="DEFAULT"):
```
Comment (rej156) [inline gcc_utils.py:27]: Breaks other python files using this function,  it doesn't affect readability or the json print on gcp with multiple args
print() has the same multiple args definition fwiw

File: top_pools_base.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import os
+import requests
+from web3 import Web3
+from gcc_utils import gccPrint
+
+TAG = "[TOP_POOLS_BASE] "
+
+COVALENT_API_KEY = os.getenv("COVALENT_API_KEY", "")
+COVALENT_ENDPOINT = f"https://api.covalenthq.com/v1/1313161554/xy=k/trisolaris/pools/?quote-currency=USD&format=JSON&key={COVALENT_API_KEY}"
```
Comment (rej156) [inline top_pools_base.py:9]: Move to constants maybe

**Comment by rej156** [comment]
https://www.delftstack.com/howto/python/tuple-to-string-python/#use-the-str-join-function-to-convert-tuple-to-string-in-python

"".join(message) in the else block maybe? @specialist0x

**Comment by rej156** [comment]
could also import logger and just use that everywhere

File: top_pools_base.py (hunk 2 above)
Comment (specialist0x) [reply top_pools_base.py:9]: Created a `covalent.py` file in utls

File: top_pools_base.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+def top_pools_base():
+    gccPrint(TAG + "Starting")
+
+    response = requests.get(COVALENT_ENDPOINT).json()
+    pools = response['data']['items']
```
[the remark this answers is not in the document]
Comment (specialist0x) [reply top_pools_base.py:15]: Added

[the remark this answers is not in the document]
Comment (rej156) [reply top_pools_base.py]: banger

**Pull request merged**